Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.py

```diff
@@ -0,0 +1,1330 @@
+    content = args.get('content')
+    if not repo:
+        repo = client.repository
+    body = {
+        'content': {
+            'raw': content
+        }
+    }
+    response = client.pull_request_comment_update_request(repo, pr_id, body, comment_id)
+    return CommandResults(
+        readable_output=f'The comment "{comment_id}" on the pull request "{pr_id}" was updated successfully',
+        outputs_prefix='Bitbucket.PullRequestComment',
+        outputs=[response],
+        raw_response=[response])
+
+
+def pull_request_comment_delete_command(client: Client, args: Dict) -> CommandResults:
+    """ Deletes a comment in a pull request.
+        Args:
+            client: A Bitbucket client.
+            args: Demisto args.
+        Returns:
+            A CommandResult object with a success message.
+    """
+    repo = args.get('repo', None)
+    pr_id = int(args.get('pull_request_id', None))
+    comment_id = args.get('comment_id', None)
+    if not repo:
+        repo = client.repository
+    response = client.pull_request_comment_delete_request(repo, pr_id, comment_id)
+    if response.status_code == 204:
+        return CommandResults(readable_output=f'The comment on pull request number {pr_id} was deleted successfully.')
+    else:
+        return CommandResults(readable_output=response.text)
+
+
+def workspace_member_list_command(client: Client, args: Dict) -> CommandResults:
+    """ Returns a list of all the members in the requested workspace.
+        Args:
+            client: A Bitbucket client.
+            args: Demisto args.
+        Returns:
+            A CommandResult object with the requested list.
+    """
+    limit = int(args.get('limit', 50))
+    page = int(args.get('page', 1))
+    check_args(limit, page)
+    page_size = min(100, limit)
+    params = {
+        'page': page,
+        'pagelen': page_size
+    }
+    response = client.workspace_member_list_request(params)
+    results = check_pagination(client, response, limit)
+    human_readable = []
+    for value in results:
+        d = {'Name': demisto.get(value, 'user.display_name'),
+             'AccountId': demisto.get(value, 'user.account_id')
+             }
+        human_readable.append(d)
+    headers = ['Name', 'AccountId']
+    readable_output = tableToMarkdown(
+        name='The list of all the workspace members',
+        t=human_readable,
+        removeNull=True,
+        headers=headers
+    )
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Bitbucket.WorkspaceMember',
+        outputs=results,
+        raw_response=results
+    )
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:  # pragma: no cover
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+
+    workspace = demisto.params().get('Workspace', "")
+    server_url = demisto.params().get('server_url', "")
+    user_name = demisto.params().get('UserName', "").get('identifier', "")
```

Comment:
use credentials for the user name and password
    username = params.get('credentials', {}).get('identifier')
    password = params.get('credentials', {}).get('password')

Reply:
According to the demistomock.py, the isentifier and password keys are in the dictionary of UserName, but not in the credentials:
UserName : {
  credentials:{...}
  identifier: '...'
  password: '...'
}
So when I run what you suggested it doesn't recognize them.